src/excel/utilities.ts

```diff
@@ -38,37 +37,79 @@ export class ExcelUtilities {
             throw new APIError('Sheet name cannot be greater than 31 characters.');
         }
 
-        const { context } = workbook;
-        const newSheet = workbook.worksheets.add();
+        options = options || {};
```

Comment:
It was intentional, in that I wanted `if (options.clearOnly)` not to blow up below.
But I suppose I can replace it with `if (options && options.clearOnly)`.  Will do so now.